roles/keystone/tasks/monitoring.yml

```diff
@@ -17,6 +17,19 @@
                args="--service keystone --criticality {{ keystone.monitoring.sensu_checks.check_keystone_api.criticality }}"
   notify: restart sensu-client
 
+- block:
+  - name: create directory for below symbolic link
+    file:
+      path: /opt/openstack/current/keystone/bin/
+      state: directory
+  
+  - name: add symolic link for keystone expired token check (rhel distro only)
+    file: 
+      src: /bin/python
+      dest: /opt/openstack/current/keystone/bin/python
+      state: link
+  when: openstack_install_method == 'distro' and os == 'rhel'
+
```

Comment:
What is this task accomplishing? Could we change the expired token check to just use the python interpreter from the sensu venv ?